iothub/device/samples/solutions/PnpDeviceSamples/PnpConvention/PnpConvention.cs

```diff
@@ -34,9 +34,8 @@ public class PnpConvention
         /// <see href="https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md#telemetry"/>.</param>
         /// <param name="telemetryValue">The unserialized telemetry payload, in the format defined in the DTDL interface.</param>
         /// <param name="componentName">The name of the component in which the telemetry is defined. Can be null for telemetry defined under the root interface.</param>
-        /// <param name="encoding">The character encoding to be used when encoding the message body to bytes. This defaults to utf-8.</param>
         /// <returns>A plug and play compatible telemetry message, which can be sent to IoT Hub. The caller must dispose this object when finished.</returns>
-        public static Message CreateMessage(string telemetryName, object telemetryValue, string componentName = default, Encoding encoding = default)
+        public static OutgoingMessage CreateMessage(string telemetryName, object telemetryValue, string componentName = default)
```

Comment:
Once all pnp changes are in, I'll delete this helper and have our sample use SDK code directly